services/device-registry-base/src/main/java/org/eclipse/hono/service/management/tenant/TenantManagementService.java

```diff
@@ -65,6 +71,43 @@
      */
     Future<OperationResult<Tenant>> readTenant(String tenantId, Span span);
 
+    /**
+     * Finds tenants with optional filters, paging and sorting options.
+     * <p>
+     * This search operation is considered as optional since it is not required for the normal functioning of Hono and
+     * is more of a convenient operation. Hence here it is declared as a default method which returns
+     * {@link HttpURLConnection#HTTP_NOT_IMPLEMENTED}. It is upto the implementors of this interface to offer an
```

Comment:
it is up to the ...

Reply:
Thanks for pointing out, I have corrected it.